src/pages/Ticketing/section/Movie/Movie.js

```diff
@@ -0,0 +1,116 @@
+import { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+const Movie = () => {
+  const [movieList, setMovieList] = useState([]);
+  const [activeMovie, setActiveMovie] = useState('');
+  const [getMovieId, setGetMovieId] = useState(0);
+
+  useEffect(() => {
+    fetch('data/movieList.json')
+      .then(response => response.json())
+      .then(result => setMovieList(result));
+  }, []);
+
+  console.log(movieList);
+  console.log(activeMovie);
+  return (
+    <MovieContainer>
+      <Title>영화</Title>
+      <Information>
+        <Top>
+          <AgeLimit>연령</AgeLimit>
+          <MovieName>제목</MovieName>
+        </Top>
+        <MovieList key={getMovieId.id}>
+          {movieList &&
+            movieList.map(item => (
```

Comment:
빈 배열은 truthy한 값이기 때문에 조건 넣어주지 않으셔도 됩니다!
```suggestion
          {movieList.map(item => (
```